operate/ols-using-openshift-lightspeed.adoc

```diff
@@ -15,8 +15,6 @@ There are different ways you can access the {ols-official} user interface.
 The following topics provide information about submitting questions to {ols-full}.
 
 include::modules/ols-initiating-chat-using-chat-window.adoc[leveloffset=+1]
-include::modules/ols-initiating-chat-using-actions-list.adoc[leveloffset=+1]
-include::modules/ols-initiating-chat-using-node-options-icon.adoc[leveloffset=+1]
```

Comment:
If these files are no longer in use by anything (which seems to be the case on this branch), then they can be fully removed/deleted from the branch. If you think they might be useful in the future, then maybe could move them to the `unused_topics/` folder for now.

Reply:
Thanks for this suggestion...as well as the notion that peer review should still be done.

Reply:
Pushed change to PR. 

Done.